app/actions/users.go

```diff
@@ -0,0 +1,175 @@
+package actions
+
+import (
+	"fmt"
+	"mjm/app/models"
+	"net/http"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/buffalo/render"
+	"github.com/gobuffalo/pop/v6"
+)
+
+func List(c buffalo.Context) error {
```

Comment:
This name is too general when we are working with several models or several CRUDs, this should be called **ListUsers**, in this way there could be several actions to list example ListDepartment, etc.